garminexport/garminclient.py

```diff
@@ -268,8 +148,8 @@ def _fetch_activity_ids_and_ts(self, start_index, max_limit=100):
         :rtype: tuples of (int, datetime)
         """
         log.debug("fetching activities %d through %d ...", start_index, start_index + max_limit - 1)
-        response = self.session.get(
-            "https://connect.garmin.com/activitylist-service/activities/search/activities",
+        response = self.gc.get(
+            "connectapi", "/activitylist-service/activities/search/activities", api=True,
```

Comment:
Some time between November 1 and today, this API call is now returning an empty list instead of my activity IDs (so I falsely have zero activities instead of almost a thousand)